validator/validator-main.protoascii

```diff
@@ -1371,6 +1371,7 @@ tags: {
       protocol: "web+mastodon"  # See GitHub issue #14793
       protocol: "wh"
       protocol: "whatsapp"
+      protocol: "microsoft-edge"
```

Comment:
Please add a comment with a link to:
https://docs.microsoft.com/en-us/windows/uwp/launch-resume/launch-default-app#microsoft-edge-uri-scheme

or something similar.

Reply:
@Gregable Sure. 

Added it :+1: .

Reply:
Looks the validation entry needs to be sorted alphabetically as well.